ruby-on-rails/ruby-on-rails-quiz.md

```diff
@@ -1059,4 +1097,53 @@ end
 - [ ] `To seed the database with initial data.`
 - [ ] `To test the database connection.`
 
-[reference](https://guides.rubyonrails.org/v3.2/migrations.html#:~:text=Active%20Record%20tracks%20which%20migrations,the%20structure%20of%20your%20database.)
+[Reference](https://guides.rubyonrails.org/v3.2/migrations.html#:~:text=Active%20Record%20tracks%20which%20migrations,the%20structure%20of%20your%20database.)
+
+#### Q79. What is the execution result of the following Ruby code?
```

Comment:
New question a little bit more ruby-related concerning how self (although consistant) differs inside class / instance methods in a Class.